cli/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "node16",
+    "target": "ES2022",
+    "declaration": true,
+    "sourceMap": true,
+    "outDir": "dist",
+    "esModuleInterop": true,
+    "moduleResolution": "nodenext",
+    "skipLibCheck": true,
+    "allowJs": false,
+    "noImplicitAny": true
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "templates"]
```

maybe exclude also the dist folder 

```suggestion
  "exclude": ["node_modules", "templates", "dist"]
```

---

`dist` is outside of `"include": ["src/**/*"]` and nothing requires it from src => no need